### PR TITLE
reduce the number of top level navigation items from 46 to 32

### PR DESCRIPTION
**Pull request by SidneyAllen**
Fixes #\<INSERT ISSUE NUMBER HERE\>

## Checklist

- [X] Up to date with `main`
- [X] All the tests are passing
  - [X] Prettier
  - [X] Spectral Lint
  - [X ] Contract Tests
    - [X] Delete all resources created in contract tests
- [X] `npm run bundle` outputs nothing suspect
- [X] `npm run postman` outputs nothing suspect

## Changes
Consolidated content to reduce the number of top level items on the left side navigation

### CODE REVIEW

**Review by rogue0137 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
...
...-templateOptions.description=\"Detailed developer documentation for Lob's address verification API and print & mail API. Learn how to easily integrate Lob's APIs within your product or technology stack.\" --templateOptions.LOB_URL=https://lob.com --templateOptions.image=https://s3-us-west-2.amazonaws.com/public.lob.com/marketing/opengraph2.png --templateOptions.DASHBOARD_URL=https://dashboard.lob.com/# --title=\"DRAFT NEW Lob API documentation\" --options.noAutoAuth=false --options.maxDisplayedEnumValues=15 --options.theme.colors.primary.main=#27333c --options.theme.sidebar.backgroundColor=#f7f9fa --options.jsonSampleExpandLevel=1 --options.requiredPropsFirst=true --options.theme.typography.code.backgroundColor=#ebf0f6 --options.theme.typography.code.fontFamily=JetBrainsMono --options.theme.typography.code.color=#27333c --options.theme.typography.fontFamily=\"Larsseit,sans-serif;\" --options.theme.typography.links.color=#0099d7 --options.theme.typography.links.textDecoration=none --options.theme.codeBlock.backgroundColor=#ecf0f5 --options.theme.rightPanel.backgroundColor=#f7f9fa --options.theme.rightPanel.textColor=#416581 --options.theme.typography.fontSize=16 -o docs/index.html",
```
Comment (jho44) [inline package.json:60]: We have 2 authentication sections now. Is this what you intended?

**Review by jho44 — CHANGES_REQUESTED** [review]
Some of the inline links are also now invalid like Cancellation WIndow

File: package.json (hunk 1 above)
Comment (SidneyAllen) [reply package.json:60]: Sadly - there is a link with each operation that points to the default "authentication" section and I can't find a way to remove the link or change it - so I added the default authentication section back in and renamed ours "basic authentication".  If you find a way around this please let me know.

Comment (jho44) [reply package.json:60]: I think I could include that in the remapping-hashes PR so u can probably get rid of the default "authentication" section and turn "basic authentication" back into "authentication". And I can map `#section/Authentication/basicAuth` to `#tag/Authentication`

Comment (SidneyAllen) [reply package.json:60]: I've also hidden the download button as I have no idea how it works, but it someone generates a file called swagger.json for download.  It's cool an all, but we are using OpenAPI not swagger so it's confusing.

We could create our own download button and link to our deployment branch of the lob-openapi repo in the future.

**Comment by SidneyAllen** [comment]
@jho44  - good catch on the inline links - I've updated those

**Review by jho44 — APPROVED** [review]

**Pull request merged**